### DISCUSSION TITLE
Can i set "plugin.toolkit.log.grpc.reporter.server_host" with a IP list?

### DISCUSSION THREAD

**Discussion by rushsky518** [apache/skywalking#7881] [Q&A]
When use GRPCLogClientAppender to report logs to skywalking cluster, "plugin.toolkit.log.grpc.reporter.server_host" can only be 
a single ip address.

**Comment by rushsky518** [comment]
agent versin 8.7.0

**Reply by wu-sheng** [reply]
For 8.7.0, I don't think so.
In the next release, this has been removed and shares with core backend service setting.

**Reply by rushsky518** [reply]
OK, thanks
